buoy/buoy.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"log"
+
+	"knative.dev/test-infra/buoy/commands"
+)
+
+func main() {
+	if err := commands.New().Execute(); err != nil {
+		log.Fatal("error during command execution: ", err)
```

Comment:
```suggestion
		log.Fatal("Error during command execution: ", err)
```
this is end user message so a complete sentence.